src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.*;
```

Comment:
According to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html), you may want to list all imports explicitly as they make the class easier to read 😀